docs/core/tools/dotnet-test.md

```diff
@@ -11,18 +11,24 @@ ms.date: 03/27/2024
 
 ## Description
 
-The `dotnet test` command builds the solution and runs the tests with either VSTest or Microsoft Testing Platform (MTP). To enable MTP, you need to add a config file named `dotnet.config` with an INI-like format located at the root of the solution or repository.
+The `dotnet test` command builds the solution and runs the tests with either VSTest or Microsoft Testing Platform (MTP). To enable MTP, you need to specify the test runner in the `global.json` file.
 
-Some examples of the `dotnet.config` file:
+Some examples of how to specify the test runner in the `global.json` file:
```

Comment:
Please also update https://learn.microsoft.com/dotnet/core/tools/global-json.
Probably it's worth to hyperlink the global.json in this sentence to link the global.json doc as well.